CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ## Unreleased
 
 ### Compatible changes
+* Removed support for serial execution of scenarios tagged with @solo.
```

Comment:
Not sure if this might not be a breaking change :thinking: 
It does not break code but it is an unexpected change in behavior. What do you think @codener

Reply:
It's tricky. Since serial execution was only ever needed for parallel tests inside VNC, and VNC support is already removed, I'd call it "compatible change". On the other hand, users tagging scenarios with `@solo` must adapt. However, obsolete `@solo` tags do no harm. Let's go with "compatible".